index.bs

```diff
@@ -352,75 +398,126 @@ interface ImageCapture {
+
+    <li><i><dfn>Contrast</dfn></i> is the numeric camera setting that controls the difference in brightness between light and dark areas in a scene.  A higher contrast setting reflects an expansion in the difference in brightness. The range and effect of this setting is implementation dependent but it can be understood as a transformation of the pixel values so that the luma range in the histogram becomes larger; the transformation is sometimes as simple as a gain factor.</li>
+
+    <li>[[LIGHTING-VOCABULARY]] defines <i><dfn>saturation</dfn></i> as "the colourfulness of an area judged in proportion to its brightness" and in the current context it refers to a numeric camera setting that controls the intensity of color in a scene (i.e. the amount of gray in the scene). Very low saturation levels will result in photos closer to black-and-white. Saturation is similar to <a>contrast</a> but referring to colors, so its implementation, albeit being platform dependent, can be understood as a gain factor applied to the chroma components of a given image.</li>
+
+    <li><i><dfn>Sharpness</dfn></i> is a numeric camera setting that controls the intensity of edges in a scene.  Higher sharpness settings result in higher contrast along the edges, while lower settings result in less contrast and blurrier edges (i.e. soft focus). The implementation is platform dependent, but it can be understood as the linear combination of an edge detection operation applied on the original image and the original image itself; the relative weights being cotrolled by this |sharpness|.
+
+    <div class="note">
+      <a>Brightness</a>, <a>contrast</a>, <a>saturation</a> and <a>sharpness</a> are specified in [[UVC]].
+    </div></li>
+
+    <li><dfn>Image width</dfn> and <dfn>image height</dfn> are...
+
+    <div class="note">
+      The supported resolutions are managed segregated e.g. {{MediaTrackCapabilities/imageWidth}} and {{MediaTrackCapabilities/imageHeight}} values/ranges to prevent increasing the fingerprinting surface and to allow the UA to make a best-effort decision with regards to actual hardware configuration vis-a-vis requested constraints.
+    </div>
+    </li>
+
+    <li><dfn>Zoom</dfn> is a numeric camera setting that controls the focal length of the lens.  The setting usually represents a ratio, e.g. 4 is a zoom ratio of 4:1.  The minimum value is usually 1, to represent a 1:1 ratio (i.e. no zoom).</li>
+
+    <li><dfn>Fill light mode</dfn> describes the flash setting of the capture device (e.g. |auto|, |off|, |on|). </li>
+  </ol>
+
+# {{MediaSettingsRange}} # {#mediasettingsrange-section}
 
 <pre class="idl">
   interface MediaSettingsRange {
```

Comment:
MediaSettingsRange is an interface 
```
<pre class="idl">
  interface MediaSettingsRange {
      readonly attribute double max;
      readonly attribute double min;
      readonly attribute double step;
  };
</pre>
```
whereas [`DoubleRange`](https://www.w3.org/TR/mediacapture-streams/#idl-def-DoubleRange) is a dictionary.
 
```
dictionary DoubleRange {
             double max;
             double min;
};
```
It makes sense to have it as a Dictionary since it's used for configuring constraints in the track; in ImageCapture it makes sense to have it as an interface because settings ranges are always composed of min-max and step.  So, they're different ways to represent related concepts and both are OK.